python/gps/algorithm/policy_opt/config.py

```diff
@@ -38,12 +38,9 @@
 POLICY_OPT_CAFFE.update(GENERIC_CONFIG)
 
 
-checkpoint_path = os.path.abspath(os.path.join(os.path.dirname(__file__),
```

Comment:
Actually, nevermind. The way that it is coded right now is best, so you never run into file conflicts and the weights are stored with the policy. Caffe should be changed to do it how tf does it, if possible.